lib/game_logic.rb

```diff
@@ -0,0 +1,138 @@
+# frozen_string_literal: false
+
+class Rules
+  attr_reader :answer
+
+  def initialize(answer)
+    @answer = answer
+  end
+
+  def display_rules
+    case answer
+    when 'yes'
+      result = "Ok, then. The board has a 3x3 grid. Each board space is represented by a number from 1 to 9, like that:
+
+                              1 | 2 | 3
+                            -----------
+                              4 | 5 | 6
+                            -----------
+                              7 | 8 | 9
+
+Each player will be represented by a mark, X and O.
+Every round, the players will choose between an available position.
+The player who suceeds in placing three of their marks in a horizontal, vertical, or diagonal row wins the game.
+Enough talk, type OK to start the game!"
+    when 'no'
+      result = "Alright, let's start the game then. Type OK to continue."
+    else
+      result = 'Please, type yes or no.'
+    end
+    result
+  end
+end
+
+class Board
+  attr_accessor :board
+  attr_reader :mark
+
+  def initialize
+    @board = ['   ', '   ', '   ', '   ', '   ', '   ', '   ', '   ', '   ']
+  end
+
+  def display_board
+    "#{board[0]}|#{board[1]}|#{board[2]}\n" \
+    "-----------\n" \
+    "#{board[3]}|#{board[4]}|#{board[5]}\n" \
+    "-----------\n" \
+    "#{board[6]}|#{board[7]}|#{board[8]}\n"
+  end
+
+  def mark_position(index, mark)
+    board[index] = ' ' + mark + ' '
+  end
+end
+
+class Player
+  attr_accessor :name, :mark
```

Comment:
**[REQUIRED]**
 The player's `name` never changes after been assigned. You can safely use `attr_reader` for it.